react/wbooks/src/app/components/SignUp/index.js

```diff
@@ -4,26 +4,21 @@ import { get } from 'lodash';
 
 import imagePlaceholder from '../../assets/wolox-logo.png';
 import InputLabel from '../InputLabel';
+import { createUser } from '../../../services/UserService';
 
 import styles from './styles.module.scss';
 import { LOGIN, SIGN_UP, FIELDS, FIELDS_DATA } from './constants';
 
 class SignUp extends Component {
-  handleSignUp = () => {
-    const { email, password, passwordConfirmation, name, lastname } = this.state;
+  state = { isError: false };
 
-    console.log({
-      user: {
-        email,
-        password,
-        /* eslint-disable camelcase */
-        password_confirmation: passwordConfirmation,
-        first_name: name,
-        last_name: lastname,
-        /* eslint-enable camelcase */
-        locale: 'en'
-      }
-    });
+  handleSignUp = async () => {
+    try {
+      const response = await createUser({ ...this.state, locale: 'en' }); // eslint-disable-line no-unused-vars
```

Comment:
why are you destructuring the state. (i really know) but the initial state should be descriptive about what variables the state will have.